transport-msg.tex

```diff
@@ -1048,6 +1048,11 @@ \subsubsection{Overview}
 \hline
 \end{tabular}
 
+If virtqueue index is 0xFFFF then it indicates that all virtqueus should be
+polled for available buffers. This is useful when the bus implementation is
+using out-of-band notifications, using polling, or when emulating or bridging
+to another virtio transport.
```

Comment:
this seems breaking the protocol as 0xFFFF is a valid index


```
2.6 Virtqueues
The mechanism for bulk data transport on virtio devices is pretentiously called a virtqueue. Each device can
have zero or more virtqueues1.
A virtio device can have maximum of 65536 virtqueues. Each virtqueue is identified by a virtqueue index. A
virtqueue index has a value in the range of 0 to 65535
```

I would remove this and add
```
Virtqueue index greater than 0xFFFF can be used by the bus for out of band notifications. For instance that  that all virtqueus should be polled for available buffers or when emulating or bridging
to another virtio transport.
```

Reply:
I meant 0xFFFF_FFFF.  Ie -1 but it is not -1 as it is not a signed int.
To be of value here there needs to be an agreed value between bus and transport.

Otherwise the loop can be in the bus and the bus can invent specific virtqueue polls. 
In this system the bus will need to look into the transport and see how many virtqueues it has.